pkg/backup/backup.go

```diff
@@ -200,7 +200,7 @@ func (bc *BackupController) Run() {
 	}
 }
 
-// SaveSnap saves the snapshot if snapshot's revision is greater than the given lastSnapRev
+// SaveSnap saves the snapshot if latest snapshot's revision is greater than the given lastSnapRev
```

Comment:
`SaveSnap saves the snapshot if latest snapshot's revision is greater than the given lastSnapRev` ->

`SaveSnap saves the latest snapshot if its revision is greater than the given lastSnapRev`?